libs/application/templates/financial-aid/src/fields/TaxReturnFilesForm/TaxReturnFilesForm.tsx

```diff
@@ -10,12 +10,22 @@ import { getTaxFormContent } from './taxFormContent'
 
 const TaxReturnFilesForm = ({ field, application }: FAFieldBaseProps) => {
   const { formatMessage } = useIntl()
-  const { id, answers, externalData } = application
+  const { id, answers, externalData, assignees } = application
 
-  const {
-    municipalitiesDirectTaxPayments,
-    municipalitiesPersonalTaxReturn,
-  } = externalData.taxDataFetch.data
+  const isSpouse = (assignees: string[], nationalId?: string) => {
```

Comment:
We have a similar check in mapUserToRole in FinancialAidTemplate.ts. Just wondering if we could create a function in utils.ts and reuse it for both places?

Reply:
changed it made it similar, what do you think?

Reply:
I was thinking of creating a function in utils (getUserRole or something like that) using the code from mapUserToRule in FinancialAidTemplate:

```
    if (id === application.applicant) {
      return Roles.APPLICANT
    }
    if (application.assignees.includes(id)) {
      return Roles.SPOUSE
    }
    return undefined
```

Then call this utils function inside of mapUserToRule and TaxReturnFilesForm.ts
How do you feel about that?

Reply:
I like how you think, however in the application system the mapUserToRole has to be in the FinancialAidTemplate, because there is some magic happening there and all other template do it the same way. And the function would return Roles.SPOUSE and then i would also have to check that as well, instead of doing the check if the nationalId is in assignees, I can add that it to utils, but i thought it was unnecessary since it is only used here, however i do agree if we need the check again we should move the function to utils.